frontend/src/components/books/BookDetails/BookDetails.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactElement } from 'react';
+import { Row, Col } from 'react-flexbox-grid';
+
+import { Book } from '../../../../../interfaces/books';
+import BookImage from './BookImage/BookImage';
+
+interface Props {
+  book: Book;
+}
+
+function BookDetails({ book }: Props): ReactElement {
+  return (
+    <div>
```

Comment:
Just a suggestion that IMO looks better 

```suggestion
const BookDetails: FC<Props> = ({ book }) => (
  <div>
```

Reply:
Sure! 👍🏼